swag/errors.py

```diff
@@ -68,3 +68,47 @@ def __init__(self, date):
         self.date = date
         message = f"Vous ne pouvez pas changer de timezone avant {date}."
         super().__init__(message)
+
+
+##Exception for €agnottes
+
+
+class NoCagnotteIdxInCommand(Exception):
+    """Raised when a user use a cagnotte command without specify a cagnotte idx with an €"""
+
+    pass
+
+
+class NoCagnotteRegistered(Exception):
+    """Raised when the cagnotte name or id is not present in the Swagbank"""
+
+    def __init__(self, name):
+        self.name = name
+        message = f"La €agnotte {name} n'existe pas dans la $wagBank"
+        super().__init__(message)
+
+    pass
+
+
+class CagnotteNameAlreadyExist(Exception):
+    """Raised when a new cagnotte is created with an already used name"""
+
+    pass
+
+
+class NotEnoughMoneyInCagnotte(Exception):
+    """Raised when a Cagnotte should have a negative value of $wag or $tyle"""
+
+    pass
+
+
+class NotInGestionnaireGroupCagnotte(Exception):
```

Comment:
naming

Reply:
Resolved in : https://github.com/BobbySoftwares/Bybbot/pull/7/commits/db86cd2b150fe220ee034b87e717cda3ee201338